tasks/main.yml

```diff
@@ -0,0 +1,27 @@
+---
+- name: Install openvpn
+  apt: 
+    name: openvpn
+
+- name: Slurp openvpn config
+  slurp:
+    src: "{{ hostvars[openvpn_client_server].openvpn_base_dir }}/{{ openvpn_client_name }}-{{ openvpn_client_server }}.ovpn"
+  register: openvpn_config_file
+  delegate_to: "{{ openvpn_client_server }}"
+  when: openvpn_client_server is defined
+
+- name: Copy to host
+  copy:
+    content: "{{ openvpn_config_file['content'] | b64decode }}"
+    dest: "/etc/openvpn/{{ openvpn_client_name }}-{{ openvpn_client_server }}.conf"
+  when: openvpn_client_server is defined
+  notify: Restart openvpn service
+
+- name: Copy to host
+  copy:
+    src: "{{ openvpn_client_config_file_name }}"
```

Comment:
In the past we used a structure in `files` that takes files from a subfolder named after the host.
Does this work for this case, too or would the path have to be specified in this role?

Reply:
fixed